docs/arch/adr-03.md

```diff
@@ -0,0 +1,112 @@
+# ADR 03: Azure Data Storage Tools
+
+* Status: Awaiting Review
+* Date: Dec 23, 2019
+
+## Context
+
+When a transcription is approved, a set of flat files containing the transcription data will be saved to Azure. Users will have the option to download a zip file containing their requested subject, group, workflow, or project. Depending on the speed at which we are able to zip the necessary files, we will either trigger a direct download, or provide a link to the location of the zip file to the user. 
+
+The goal is to investigate Azure’s storage options (specifically Blob Storage and File Services) and decide which tool is best suited for our needs.
+
+### Factors to consider:
+
+* How easy is it to share a file to the end user? What is the process for this?
```

Comment:
Still TBD, as far as I can tell. The access signature method sounds completely the same as S3, but we'll have to figure out when/where they should be generated and sent to the user.

Reply:
True. Makes sense to leave this as part of the next step(s).